### PR TITLE
[WIP] Edit notice file

### PR DESCRIPTION
**Pull request by kleinech**
Edit notice file

- Add third party libraries excel sheet

Signed-off-by: Christoph Kleine <kleinech.github@gmail.com>

#### Short Description
TODO: 
- [ ] Validate Copyright Holders dates 
- [ ] Complete list of Copyright Holders

### CODE REVIEW

**Comment by miwurster** [comment]
Please add `Copyright 2017 Michael Wurster`.

File: NOTICE [hunk 1]
```diff
@@ -1,46 +1,619 @@
...
+== Source Code
+
+* https://github.com/OpenTOSCA/container
+
+== Copyright Holders
+
+Copyright (c) 2012-2017 University of Stuttgart
+Copyright (c) 2014-2015 Tobias Binz
+Copyright (c) 2012-2017 Uwe Breitenbücher
+Copyright (c) 2012-2017 Florian Haupt // TODO Check
```
Comment (F7502) [inline NOTICE]: OK

File: NOTICE [hunk 2]
```diff
@@ -1,46 +1,619 @@
...
+Copyright (c) 2012-2017 Oliver Kopp
+Copyright (c) 2012-2017 Alexander Nowak // TODO Check
+Copyright (c) 2012-2017 Sebastian Wagner // TODO Check
```
Comment (F7502) [inline NOTICE]: Left the university end of 2016.

File: NOTICE [hunk 3]
```diff
@@ -1,46 +1,619 @@
...
+* https://github.com/OpenTOSCA/container
+
+== Copyright Holders
+
+Copyright (c) 2012-2017 University of Stuttgart
+Copyright (c) 2014-2015 Tobias Binz
+Copyright (c) 2012-2017 Uwe Breitenbücher
+Copyright (c) 2012-2017 Florian Haupt // TODO Check
+Copyright (c) 2012-2017 Oliver Kopp
+Copyright (c) 2012-2017 Alexander Nowak // TODO Check
```
Comment (F7502) [inline NOTICE]: Left the university end of 2014.

File: NOTICE [hunk 4]
```diff
@@ -1,46 +1,619 @@
...
+Copyright (c) 2012-2017 Sebastian Wagner // TODO Check
+Copyright (c) 2012-2017 Tim Waizenegger // TODO Check
+Copyright (c) 2012-2017 Marcus Eisele  // TODO Check
```
Comment (F7502) [inline NOTICE]: Member of first StuPro 2012-2013 + Bachelor thesis (instance data API) until end of 2013.

File: NOTICE [hunk 5]
```diff
@@ -1,46 +1,619 @@
...
+Waizenegger at the Institute of Architecture of Application Systems (IAAS) and
+the Institute for Parallel and Distributed Systems - Applications of Parallel
+and Distributed Systems (IPVS/AS) of the University of Stuttgart.
+For more information see http://www.opentosca.org
+
+This work has been supported by the Federal Ministry of Economics and
+Technology (http://www.bmwi.de/EN/root.html) as part of the CloudCycle
+project (01MD11023).
+For more information see http://www.cloudcycle.org/en/
+
```
Comment (F7502) [inline NOTICE]: Please also add Migrate! project here, similar as CloudCycle:
- also supported by BMWi
- project name: Migrate!
- funding number: 01ME11055
- website: http://www.migrate-it2green.de/

File: NOTICE [hunk 6]
```diff
@@ -1,46 +1,619 @@
...
+Copyright (c) 2012-2017 Uwe Breitenbücher
+Copyright (c) 2012-2017 Florian Haupt // TODO Check
+Copyright (c) 2012-2017 Oliver Kopp
+Copyright (c) 2012-2017 Alexander Nowak // TODO Check
+Copyright (c) 2012-2017 Sebastian Wagner // TODO Check
+Copyright (c) 2012-2017 Tim Waizenegger // TODO Check
+Copyright (c) 2012-2017 Marcus Eisele  // TODO Check
+Copyright (c) 2012-2017 Kálmán Képes // TODO Check
+Copyright (c) 2012-2017 Michael Zimmermann // TODO Check
+Copyright (c) 2012-2017 Rene Trefft // TODO Check
```
Comment (F7502) [inline NOTICE]: Member of first StuPro 2012-2013 + Bachelor thesis in 2013.

File: NOTICE [hunk 7]
```diff
@@ -1,46 +1,619 @@
...
+Copyright (c) 2012-2017 Christian Endres // TODO Check
+Copyright (c) 2012-2017 Matthias Fetzer // TODO Check
```
Comment (F7502) [inline NOTICE]: Member of first StuPro 2012-2013.

File: NOTICE [hunk 8]
```diff
@@ -1,46 +1,619 @@
...
+Copyright (c) 2012-2017 Markus Fischer // TODO Check
+Copyright (c) 2012-2017 Nedim Karaoğuz // TODO Check
```
Comment (F7502) [inline NOTICE]: Member of first StuPro 2012-2013.

**Comment by miwurster** [comment]
@kleinech Who is working on this PR?
What's necessary to finalize this activity?

**Comment by koppor** [comment]
@F7502 Thank you for the review. I think, the list of authors is complete. If not, please directly edit NOTICE in the master.

**Pull request merged**